index.js

```diff
@@ -112,6 +135,61 @@ class Conf {
 		}
 	}
 
+	_migrate(migrations, versionToMigrate) {
+		let previousMigratedVersion = this._get(MIGRATION_KEY, '0.0.0');
+
+		const newerVersions = Object.keys(migrations)
+			.filter(candidateVersion => this._shouldPerformMigration(candidateVersion, previousMigratedVersion, versionToMigrate))
+			.sort(semver.compare);
+
+		let storeBackup = {...this.store};
+
+		newerVersions.forEach(version => {
```

Comment:
Use a `for-of` loop. (Just my preference)

Reply:
Done :)